test/integration/codebuild/buildspec.os.ubuntu.ts.yml

```diff
@@ -0,0 +1,97 @@
+version: 0.2
```

Comment:
It seemed overkill to add it to the each of the build matrices, so I created a new spec that will test *.ts in just one configuration